### DISCUSSION TITLE
Apple shortcut

### DISCUSSION THREAD

**Discussion by DustinThal** [daya0576/beaverhabits#50] [Show and tell]
Since this project is exactly what I was looking for and I couldn’t find a decent app for my iPhone, I’ve now created a shortcut that I’d like to share.

You can expand it, for example, to track via NFC.

https://www.icloud.com/shortcuts/bc2e81c8fb87417f814c3e216b4ae0cf

**Comment by daya0576** [comment]
Thank you, it looks awesome and fun :) 

It would also be great if the habits list could be fetched automatically, not sure if it is possible with shortcuts. I will definitely do some research and try to create more workflows.

**Reply by DustinThal** [reply]
Definitely possible, but since I ran into a bug on my Mac, I gave up for now. I’ll implement it tomorrow. If you can wait..

**Reply by DustinThal** [reply]
new version that takes the list over api, not sure why I the order is different, apiece gives list in right order, shortcut on some part change it...
https://www.icloud.com/shortcuts/314d3dd7191c4ac6935f3b8ea146d20f
